sample-node-api.yml

```diff
@@ -5,7 +5,7 @@ services:
       description: Sample Node API
       catalogUiTileId: sample-node-api
       instanceBaseUrls:
-        - http://mymainframe.ibm.com:18000/
+        - http://${ZOWE_EXPLORER_HOST}:${GATEWAY_PORT}/
```

Comment:
`${GATEWAY_PORT}` should remain 18000 here, should be the port of the current service.